src/main/java/pl/senla/hotel/utils/HotelServiceReadDtoMapperUtil.java

```diff
@@ -0,0 +1,109 @@
+        Integer idOrder = hotelServiceReadDto.getIdOrder();
+        Integer idRoom = hotelServiceReadDto.getIdHotelFacility();
+        Optional<HotelService> hotelServiceDB = serviceRoomReservation.findServicesForOrder(idOrder).stream()
+                .filter(hs -> hs.getRoom().getIdRoom().equals(idRoom))
+                .findFirst();
+        if(hotelServiceDB.isPresent()) {
+            idService = hotelServiceDB.get().getIdService();
+        }
+        hotelService.setIdService(idService);
+        hotelService.setOrder(serviceOrder.read(idOrder));
+        hotelService.setTypeOfService(TypeOfService.valueOf("ROOM_RESERVATION"));
+        hotelService.setGuest(serviceGuest.read(hotelServiceReadDto.getIdGuest()));
+        hotelService.setRoom(serviceFacility.read(idRoom));
+        hotelService.setNumberOfDays(hotelServiceReadDto.getNumberOfDays());
+        hotelService.setCheckInTime(convertStringToTime(hotelServiceReadDto.getCheckInTimeString()));
+        hotelService.setCheckOutTime(convertStringToTime(hotelServiceReadDto.getCheckOutTimeString()));
+        hotelService.setCost(hotelServiceReadDto.getCost());
+        return hotelService;
+//        return HotelService.builder()
+//                .idService(idService)
+//                .order(serviceOrder.read(idOrder))
+//                .typeOfService(TypeOfService.valueOf("ROOM_RESERVATION"))
+//                .guest(serviceGuest.read(hotelServiceReadDto.getIdGuest()))
+//                .room(serviceFacility.read(idRoom))
+//                .numberOfDays(hotelServiceReadDto.getNumberOfDays())
+//                .checkInTime(convertStringToTime(hotelServiceReadDto.getCheckInTimeString()))
+//                .checkOutTime(convertStringToTime(hotelServiceReadDto.getCheckOutTimeString()))
+//                .cost(hotelServiceReadDto.getCost())
+//                .build();
+    }
+
+    public static HotelServiceReadDto convertHotelServiceToHotelServiceDto(HotelService hotelService) {
+        log.debug("Start: convert HotelService to HotelServiceDto");
+        return HotelServiceReadDto.builder()
+                .idOrder(hotelService.getOrder().getIdOrder())
+                .idGuest(hotelService.getGuest().getIdGuest())
+                .idHotelFacility(hotelService.getRoom().getIdRoom())
+                .numberOfDays(hotelService.getNumberOfDays())
+                .checkInTimeString(convertTimeToString(hotelService.getCheckInTime()))
+                .checkOutTimeString(convertTimeToString(hotelService.getCheckOutTime()))
+                .cost(hotelService.getCost())
+                .build();
+    }
+
+    public static List<HotelServiceReadDto> convertListHotelServiceToHotelServiceDto(List<HotelService> hotelServices) {
+        return hotelServices.stream()
+                .map(HotelServiceReadDtoMapperUtil::convertHotelServiceToHotelServiceDto)
+                .toList();
+    }
+
+    public static LocalDateTime convertStringToTime(String checkedDateString) {
```

Comment:
spring will do it automatically, just put the necessary type in consuming DTO, but you will have to provide it in the standard form